subprojects/test-runner/test-report/src/main/kotlin/com/avito/android/test/report/transport/TransportMappers.kt

```diff
@@ -79,8 +80,15 @@ interface TransportMappers {
     }
 
     private fun Any.getFieldStringValue(field: Field): String {
+        val fieldNameCapitalized = field.name.replaceFirstChar {
+            if (it.isLowerCase()) {
+                it.titlecase(Locale.getDefault())
+            } else {
+                it.toString()
+            }
```

Comment:
We could reuse here capitalize function

Reply:
nope, this module doesn't depend on `gradle-extensions' and we dont't have `kotlin-extensions` and i feel bad about creating one only for single function

Reply:
Why `Strings.kt` is in gradle module? 
It doesn't depend on Gradle and can be in a common pure kotlin module ("lang" as possible name).

As for me, I would merge also merge `result` module to this common lang module. It is also only about the Kotlin itself.

Reply:
I don't think it's a good idea to expose this capitalize function to all possible clients; same for Result

Reply:
Didn't get it, but don't want to argue in comments